okteto.yml

```diff
@@ -0,0 +1,14 @@
+name: python-getting-started
+build:
+  hello-world:
+    image: okteto.dev/python-hello-world:1.0.0
+    context: .
+deploy:
+  - kubectl apply -f k8s.yml
+dev:
+  hello-world:
+    command: python app.py
+    sync:
+      - .:/usr/src/app
+    forward:
```

Comment:
remove `forward`, we are not exposing app ports in the samples.
in python, the debugger is configured with a reverse tunnel:
```
reverse:
  - 9000:9000
```